pyface/color.py

```diff
@@ -0,0 +1,260 @@
+# (C) Copyright 2005-2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+""" Color classes and corresponding trait types for Pyface.
+
+The base Color class holds red, green, blue and alpha channel values as
+a tuple of normalized values from 0.0 to 1.0.  Various property traits
+pull out the individual channel values and supply values for the HSV
+and HSL colour spaces (with and without alpha).
+
+The ``from_toolkit`` and ``to_toolkit`` methods allow conversion to and
+from native toolkit color objects.
+"""
+
+import colorsys
+
+from traits.api import (
+    HasStrictTraits, Property, Range, Tuple, cached_property
+)
+
+
+def channels_to_ints(channels, maximum=255):
```

Comment:
Looks like we never use a different value for `maximum`. When do we need to set a different value?

Reply:
We do in the parsing code in the other PR.

Reply:
OK